{"name": "test_authentication_error_anthropic", "status": "passed", "description": "Test Anthropic authentication error produces standard format.", "attachments": [{"name": "log", "source": "134af916-c94a-4141-8dcb-4e7487641714-attachment.txt", "type": "text/plain"}], "start": 1757656635306, "stop": 1757656635306, "uuid": "b91dc4ad-b384-4ae3-a100-acf0c9312a77", "historyId": "4cf8759afeb15f6d713e8628096554ed", "testCaseId": "4cf8759afeb15f6d713e8628096554ed", "fullName": "tests.unit.test_standardized_llm_errors.TestStandardizedErrors#test_authentication_error_anthropic", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit"}, {"name": "suite", "value": "test_standardized_llm_errors"}, {"name": "subSuite", "value": "TestStandardizedErrors"}, {"name": "host", "value": "Joshuas-MacBook-Pro-2.local"}, {"name": "thread", "value": "29528-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.test_standardized_llm_errors"}], "titlePath": ["tests", "unit", "test_standardized_llm_errors.py", "TestStandardizedErrors"]}